CHANGELOG.md

```diff
@@ -5,14 +5,15 @@ All notable changes to this project will be documented in this file.
 ## [Unreleased]
 
 ### Added
-- [#1213](https://github.com/pints-team/pints/pull/1213) Added the truncated normal distribution as a log prior, `TruncatedNormalLogPrior`
+- [#1213](https://github.com/pints-team/pints/pull/1213) Added the truncated normal distribution as a log prior. As of [#1216](https://github.com/pints-team/pints/pull/1216), it is called `TruncatedGaussianLogPrior`.
```

Comment:
How about we put both PRs in front (so #1213, #1216) and then ignore the fact that it was called Normal at any point?

Seems to me the changelog should only document things changed since the last release

Reply:
That makes sense, I've made an entry just for adding truncated Gaussian